contributing-guides/style-guide.md

```diff
@@ -104,40 +104,60 @@ Example:
 Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 ```
 
-## Token syntax
+## Placeholder syntax
 
-User-provided values should use the `{{token}}` syntax
+User-provided values should use the `{{placeholder}}` syntax
 in order to allow `tldr` clients to highlight them.
 
-Keep the following guidelines in mind when choosing tokens:
+### General rules
 
-### Naming
-- Use short but descriptive tokens,
-  such as `{{source_file}}` or `{{wallet.txt}}`.
-- Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word tokens.
+- Use short but descriptive placeholders,
+  such as `{{path/to/source.cs}}` or `{{wallet.txt}}`.
+- Use [snake_case][snake_case] for multi-word tokens.
```

Comment:
Did you remove the URL on purpose? I'm not sure if `[...][...]` is even valid markdown.